### PR TITLE
add cmd get, delete, keys

### PR DESCRIPTION
**Pull request by Moeen89**
#115  , I will be happy to help if there is a problem. also I handled using put command before start.

### CODE REVIEW

File: cmd/data.go [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+	if db == nil {
+		fmt.Println("start server first")
+		return nil
+	}
+	value, err := db.Get([]byte(key))
+	if err != nil {
+		fmt.Println("get data error: ", err)
+		return err
+	}
+	fmt.Println(string(value[:]))
```
Comment (sjcsjc123) [inline cmd/data.go:43]: map be it can replace `value[:]` with `value`?

File: cmd/data.go [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+		fmt.Println("key is empty")
+		return nil
+	}
+	if db == nil {
+		fmt.Println("start server first")
+		return nil
+	}
+	err := db.Delete([]byte(key))
+	if err != nil {
+		fmt.Println("get data error: ", err)
```
Comment (sjcsjc123) [inline cmd/data.go:59]: delete data?

File: cmd/data.go [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+}
+
+func getKeys(c *grumble.Context) error {
+	if db == nil {
+		fmt.Println("start server first")
+		return nil
+	}
+	list := db.GetListKeys()
+	for i, bytes := range list {
+		fmt.Println(i+1, "- ", string(bytes[:]))
```
Comment (sjcsjc123) [inline cmd/data.go:73]: The display information may be optimized and more detailed

**Comment by Moeen89** [comment]
Is it okay now? If it's not, please let me know what is the problem.

**Comment by sjcsjc123** [comment]
thanks

**Pull request merged**